common/test/acceptance/tests/video/test_video_module.py

```diff
@@ -226,13 +226,28 @@ def test_cc_button_wo_english_transcript(self):
         self.navigate_to_video()
         self.video.show_captions()
 
-        # Verify that we see "好 各位同学" text in the captions
+        # Verify that we see "好 各位同学" text in the transcript
         unicode_text = "好 各位同学".decode('utf-8')
         self.assertIn(unicode_text, self.video.captions_text)
 
-    def test_cc_button_transcripts_and_sub_fields_empty(self):
+    def test_cc_button(self):
+        """
+        Scenario: CC button works correctly with transcript in YouTube move
+        Given the course has a vieo component in "Youtube" mode
+        And I have defined a transcript for the video
+        Then I see the closed captioning element over the video
+        """
+        data = {'transcripts': {'zh': 'chinese_transcripts.srt'}}
+        self.metadata = self.metadata_for_mode('youtube', data)
+        self.assets.append('chinese_transcripts.srt')
+        self.navigate_to_video()
+        self.video.show_closed_captions()
+
+        self.assertTrue(self.video.is_closed_captions_visible)
```

Comment:
This will probably need a "wait" to make sure this isn't flaky. It looks like you could probably use `self.video.wait_for_closed_captions`.

Could you also add a couple lines after this to make sure that hiding the captions works. (`self.video.hide_closed_captions()` then check that the captions are no longer visible)